plugin/trino-base-jdbc/src/main/java/io/trino/plugin/jdbc/JdbcConnectorFactory.java

```diff
@@ -63,14 +54,16 @@ public String getName()
     public Connector create(String catalogName, Map<String, String> requiredConfig, ConnectorContext context)
     {
         requireNonNull(requiredConfig, "requiredConfig is null");
+        requireNonNull(module, "module is null");
         checkSpiVersion(context, this);
 
         Bootstrap app = new Bootstrap(
                 binder -> binder.bind(TypeManager.class).toInstance(context.getTypeManager()),
                 binder -> binder.bind(NodeManager.class).toInstance(context.getNodeManager()),
                 binder -> binder.bind(VersionEmbedder.class).toInstance(context.getVersionEmbedder()),
-                new JdbcModule(catalogName),
-                moduleProvider.getModule(catalogName));
+                binder -> binder.bind(CatalogName.class).toInstance(new CatalogName(catalogName)),
```

Comment:
This line and the lines around it strongly suggest we should consider moving the CatalogName into ConnectorContext. (Just from a consistency perspective - I don't know if it's technically possible or not though).

No change requested.